packages/liveblocks-devtools/turbo.json

```diff
@@ -2,12 +2,16 @@
   "$schema": "https://turborepo.org/schema.json",
   "extends": ["//"],
   "pipeline": {
+    "build": {
+      "outputs": ["dist/chrome-**"]
```

Comment:
I've specified the exact build outputs for these commands, because they were interfering with one another in Turbo. For example, when you built Firefox, then Chrome, then Turborepo would think that the last successful build from Chrome included the Firefox assets, and restore them when re-running `turbo build`, overwriting the last successful Firefox output 😬